skycultures/tikuna/description.en.utf8

```diff
@@ -0,0 +1,39 @@
+<p align="justify">The Tikuna live in riverside villages along the Amazonas-Solimões and its tributaries, called igarapés. Their daily life revolve around the cultivation cycles of different agricultural goods; specially, manioc, used to produce the flour the Tikuna sell in the markets of the region’s main cities. They are also fishers, hunters, and gatherers of the forest products. The women make hammocks, bags, and other objects with plant fibers; and the men paint on tururi fiber and fabricate arrows, paddles, and canoes.</p>
+<p align="justify">The Tikuna live in riverside villages along the Amazonas-Solimões and its tributaries, called igarapés. Their daily life revolve around the cultivation cycles of different agricultural goods; specially, manioc, used to produce the flour the Tikuna sell in the markets of the region’s main cities. They are also fishers, hunters, and gatherers of the forest products. The women make hammocks, bags, and other objects with plant fibers; and the men paint on tururi fiber and fabricate arrows, paddles, and canoes.</p>
+<p align="justify">It should be noted that, in Brazil, the Tikuna Indians live in indigenous lands recognized by Brazil’s indigenous policy. They are located in “terrestrial islands” among non-indigenous territories occupied by the Brazilian Population, from Tabatinga to Manaus (As illustration seen in Brazilian Tikuna Atlas).</p>
+<h3 align="center">The Astronomic Correlations</h3>
+<p align="justify">According to Vieira’s interpretation, the form of Baweta (the Turtle Collective) is associated with the stars witThe “V” shape that Western astronomy relates to the head of Taurus is called Coyatchicüra (Caiman’s Jaw). Wücütcha’s Leg corresponds to the stars that groups as Orion. Its toes are Orion’s Belt and the bend of the legs are ι (iota) Orionis; κ (capa) Orionis; η (eta) Leporis and δ (delta) Leporis. </p>
+<p align="justify">During the summer, the fight between the Anteater (Tchatü) and the Jaguar (Ai) occurs. The end of the battle coincides with the death and ascension of Wücütcha. The eyes of the Anteater are located in the β (beta) of the Triangulum Australe; his claw  and body is in the constellation Norma e Ara. The right foot of the Jaguar is in Scorpio’s α (Antares), σ (Sigma), and τ (tau); its eyes are in ε (épsilon) and μ (mi), also in Scorpio; and its left paw is in λ (Shaula), υ (ípsilon Sco – Lesath) and κ (Capa) of the same constellation. Wücütcha (the Jaguar’s leg) represents the beginning of the rains in late September; and its reappearance to the east follows the setting of Tchatü in the west, at nightfall, in the end of November. Those correlations are illuminated by indigenous testimony reproduced bellow:</p>
```

Comment:
two more typos: 

- "ípsilon" must be "upsilon"
-  "bellow" should be "below"

Reply:
ok I'm already making the changes